### PR TITLE
PIP button

### PR DESCRIPTION
**Pull request by maxvektor**
- [x] change svg for PIP button

### CODE REVIEW

File: src/modules/ui/bottom-block/bottom-block.view.ts [hunk 1]
```diff
@@ -203,6 +209,14 @@ class BottomBlockView extends View<IBottomBlockViewStyles>
     this._$rootElement.classList.remove(this.styleNames.downloadButtonHidden);
   }
 
+  hidePictureInPictureButton() {
+    this._$rootElement.classList.add(this.styleNames.downloadButtonHidden);
```
Comment (maxvektor) [inline src/modules/ui/bottom-block/bottom-block.view.ts]: downloadButtonHidden -> PIP button

File: src/modules/ui/controls/pictureInPicture/picture-in-picture.view.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    this._$togglePictureInPictureControl.focus();
+    this._callbacks.onButtonClick();
+  }
+
+  setPictureInPictureState(isPictureInPicture: boolean) {
+    if (isPictureInPicture) {
+      this._$togglePictureInPictureControl.classList.add(
+        this.styleNames.inFullScreen,
+      );
+      this._$togglePictureInPictureControl.innerHTML = exitFullScreenIconTemplate(
```
Comment (maxvektor) [inline src/modules/ui/controls/pictureInPicture/picture-in-picture.view.ts]: rename exitFullScreenIconTemplate

File: src/modules/ui/controls/pictureInPicture/picture-in-picture.view.ts [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+      );
+
+      this._tooltipReference.setText(
+        this._textMap.get(TextLabel.EXIT_PICTURE_IN_PICTURE_TOOLTIP),
+      );
+    } else {
+      this._$togglePictureInPictureControl.classList.remove(
+        this.styleNames.inFullScreen,
+      );
+      this._$togglePictureInPictureControl.innerHTML = enterFullScreenIconTemplate(
```
Comment (maxvektor) [inline src/modules/ui/controls/pictureInPicture/picture-in-picture.view.ts]: rename

**Comment by kupriyanenko** [comment]
@BrooklynKing @maxvektor let's squash commits like this in the future. And use conventional commits.

**Pull request merged**